### ISSUE TITLE
[NPC]Spitelash Siren's#6195 [Spell]Frost Nova usage differ from retail

### ISSUE THREAD

**Issue by Tenoutoften** [Atlantiss/NetherwingBugtracker#1651]
**Description**: 
[[NPC]Spitelash Siren#6195](https://www.wowhead.com/npc=6195/spitelash-siren#comments) should primarily cast [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) at low health, and it's only a chance for it to do so, will very rarely use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) at high health.
--------------------
**Short fights:**
https://youtu.be/FFx7pDxRRrE?t=476 ~25% hp
https://youtu.be/FFx7pDxRRrE?t=543 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=591 ~35% hp
https://youtu.be/FFx7pDxRRrE?t=715 ~40% hp
https://youtu.be/FFx7pDxRRrE?t=752 ~40% hp
https://youtu.be/FFx7pDxRRrE?t=797 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=929 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=1127 ~25% hp
https://youtu.be/FFx7pDxRRrE?t=1286 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=1336 ~5-10% hp
https://youtu.be/FFx7pDxRRrE?t=1477 ~5-10% hp
https://youtu.be/FFx7pDxRRrE?t=1646 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=1716 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)
https://youtu.be/FFx7pDxRRrE?t=1855 ~25% hp
https://youtu.be/FFx7pDxRRrE?t=1916 ~80% hp
https://youtu.be/FFx7pDxRRrE?t=2034 - Doesn't use [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)

-------------------------
**Long fight:**
https://youtu.be/FFx7pDxRRrE?t=1046
* https://youtu.be/FFx7pDxRRrE?t=1063 looks like [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) can be cast once it reaches 50%
* https://youtu.be/FFx7pDxRRrE?t=1075 The fight has been going for roughly 30 seconds, and the other [[NPC]Spitelash Siren#6195](https://www.wowhead.com/npc=6195/spitelash-siren#comments) at full health casts [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) as he is passing it.
--------------------------
![e2a49475c12e93385a5ab4370d05ba63](https://user-images.githubusercontent.com/34839199/47070430-72507880-d1f1-11e8-97ef-81160e103bd6.png)

Current behavior on Netherwing:
https://www.youtube.com/watch?v=bZahLjBmUDM&feature=youtu.be
[[NPC]Spitelash Siren#6195](https://www.wowhead.com/npc=6195/spitelash-siren#comments) often casts [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) very early on in the fight.

**Expected behaviour**: 
In short fights: Have a chance to cast [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova) at 50% and lower, checking several times, at high health it will rarely cast [[Spell]Frost Nova](https://db.excalibur.ws/?search=frost+nova)

**Server Revision**: 
2186

**Comment by Easyfraggy** [comment]
This is incorrect. What you are noticing is a false positive - The fact that it usually happens around low hp or not at all is more likely the fact that the initial cooldown is somewhat lower on Netherwing than on retail.

We had some videos during closed beta that showed murlocs near southshore casting frost nova at high health because we also had same theory as you at one point.

**Comment by Easyfraggy** [comment]
https://youtu.be/FFx7pDxRRrE?t=1075 The fight has been going for roughly 30 seconds, and the other [NPC]Spitelash Siren#6195 at full health casts [Spell]Frost Nova as he is passing it.

From his own post literally disproves the theory.

**Comment by Tenoutoften** [comment]
At second glance it looks like they can use it at high health, I found another example. Though this seems to happen rarely: 1 was in a long fight, and the only time it casted it early in the fight at high health was at https://youtu.be/FFx7pDxRRrE?t=1916 ~80% hp 1 out of 17 attempts

**Comment by Easyfraggy** [comment]
This is likely like I said from the very start confirmation bias. IF it was hp based you would never see them use it at full health. 

Frost nova is conditioned by someone being in melee range with the mob, which might not always be the case when looking at a hunter video. 

It is possible that they have a very variable initial cooldown (say 4-20s) which would make it mostly happen late in the fight, sometimes never and sometimes very early.

**Comment by Tenoutoften** [comment]
Wouldn't really say it is confirmation bias since I don't have any opinion what the mobs will do when. I mostly rely on videos and comments - I certainly don't remember any of this from 14 years ago. But when I very often get frost nova'ed as the first move, yet it seems to rarely happen with the evidence I got, I have to present it...

I'd personally prefer to keep it as hard and brutal as possible, and this would be a nerf.

**Comment by Easyfraggy** [comment]
Yes I only object to the health based thing - because it's clearly not. So it must be the timer being very variable because theres a video of her casting it after 5 seconds in combat.

**Comment by Evolvee** [comment]
rev Block
